modules/nf-core/gatk4/postprocessgermlinecnvcalls/meta.yml

```
@@ -0,0 +1,66 @@
+name: "gatk4_postprocessgermlinecnvcalls"
+description: Postprocesses the output of GermlineCNVCaller and generates VCFs and denoised copy ratios
+keywords:
+  - gatk4
+  - postprocessgermlinecnvcalls
+  - copy number
+tools:
+  - gatk4:
+      description: |
+        Developed in the Data Sciences Platform at the Broad Institute, the toolkit offers a wide variety of tools
+        with a primary focus on variant discovery and genotyping. Its powerful processing engine
+        and high-performance computing features make it capable of taking on projects of any size.
+      homepage: https://gatk.broadinstitute.org/hc/en-us
+      documentation: https://gatk.broadinstitute.org/hc/en-us/articles/360037593411-PostprocessGermlineCNVCalls
+      doi: 10.1158/1538-7445.AM2017-3590
+      licence: ["Apache-2.0"]
+
+input:
+  - meta:
+      type: map
+      description: |
+        Groovy Map containing sample information
+        e.g. [ id:'test', single_end:false ]
+  - ploidy_model:
```

```suggestion
  - ploidy:
```
Let's try to keep the names consistent with the main.nf file